apps/deploy-web/tests/fixture/test-env.config.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from "zod";
+
+export const testEnvSchema = z.object({
+  BASE_URL: z.string().default("http://localhost:3000"),
+  TEST_WALLET_MNEMONIC: z.string()
+});
+
+export const testEnvConfig = testEnvSchema.parse({
```

Comment:
I think this is running in node env, so you can safely do
```suggestion
export const testEnvConfig = testEnvSchema.parse(process.env)
```